src/openklant/components/klantinteracties/admin/partijen.py

```diff
@@ -23,6 +24,28 @@
 )
 
 
+class PartijAdminForm(forms.ModelForm):
+    class Meta:
+        model = Partij
+        fields = "__all__"
+
+    def clean(self):
+        cleaned_data = super().clean()
+
+        voorkeurs_digitaal_adres = cleaned_data.get("voorkeurs_digitaal_adres")
+
+        if voorkeurs_digitaal_adres:
+            if (
+                voorkeurs_digitaal_adres
+                not in DigitaalAdres.objects.filter(partij=self.instance).all()
+            ):
+                raise ValidationError(
+                    _("Het voorkeurs adres moet een gelinkte digitaal adres zijn.")
+                )
```

Comment:
Adding `voorkeurs_digitaal_adres` to the `ValidationError` specifies the field that gives error
```suggestion
                raise ValidationError(
                    {
                        "voorkeurs_digitaal_adres": _(
                            "Het voorkeurs adres moet een gelinkte digitaal adres zijn."
                        )
                    }
                )
```